tidb-cloud/premium/tidb-cloud-auditing-premium.md

```diff
@@ -0,0 +1,282 @@
+
+1. In the **Bucket URI** field, enter the URI of your OSS bucket. For example, ``oss://tidb-cloud-audit-log``.
+2. In the **Bucket Region** field, select the Alibaba Cloud region where the bucket locates (recommended to match your TiDB instance region).
+3. In the **Role ARN** field, paste the Role ARN value copied in [Step 2. Configure the OSS access](#step-2-configure-oss-access).
+4. Click **Test Connection** to verify whether TiDB Cloud can access and write to the OSS bucket.
+
+    - If successful, **The connection is successful** is displayed.
+    - If not, check the OSS bucket permissions, RAM role configuration, and policy.
+
+5. Click **Enable** to activate audit logging for the instance.
+
+    TiDB Cloud is ready to write audit logs for the specified instance to your OSS bucket.
+
+> **Note:**
+>
+> - After enabling audit logging, if you make any new changes to the bucket URI or location, you must click **Test Connection** again to verify that TiDB Cloud can connect to the bucket. Then, click **Enable** to apply the changes.
+> - To remove TiDB Cloud's access to your OSS bucket, delete the trust policy granted to this instance in the Alibaba Cloud console.
+
+## Specify auditing filter rules
+
+After enabling audit logging, you must specify auditing filter rules to control which user access events to capture and write to audit logs. If no filter rules are specified, TiDB Cloud does not log anything.
+
+To specify auditing filter rules for an instance, take the following steps:
+
+1. On the **DB Audit Logging** page, click **Add Filter Rule** in the **Log Filter Rules** section to add an audit filter rule.
+
+    You can add one audit rule at a time. Each rule specifies a user expression, database expression, table expression, and access type. You can add multiple audit rules to meet your auditing requirements.
+
+2. In the **Log Filter Rules** section, click **>** to expand and view the list of audit rules you have added.
+
+> **Note:**
+>
+> - The filter rules are regular expressions and case-sensitive. If you use the wildcard rule `.*`, all users, databases, or table events in the instance are logged.
+> - Because audit logging consumes instance resources, be prudent when specifying filter rules. To minimize the consumption, it is recommended that you specify filter rules to limit the scope of audit logging to specific database objects, users, and actions, where possible.
+
+## View audit logs
+
+By default, TiDB Cloud stores database audit log files in your storage service, so you need to read the audit log information from your storage service.
+
+TiDB Cloud audit logs are readable text files with the instance ID, Pod ID, and log creation date incorporated into the fully qualified filenames.
+
+For example, `13796619446086334065/tidb-0/tidb-audit-2022-04-21T18-16-29.529.log`. In this example, `13796619446086334065` indicates the instance ID and `tidb-0` indicates the Pod ID.
```

Comment:
For premium TiDB, the audit log does not name as `tidb-0`, besides don't use `Pod` here.